src/metax_api/tests/api/rest/base/serializers/serializer_utils.py

```diff
@@ -0,0 +1,100 @@
+# This file is part of the Metax API service
+#
+# Copyright 2017-2018 Ministry of Education and Culture, Finland
+#
+# :author: CSC - IT Center for Science Ltd., Espoo Finland <servicedesk@csc.fi>
+# :license: MIT
+
+from django.core.management import call_command
+from rest_framework.serializers import ValidationError
+from rest_framework.test import APITestCase
+
+from metax_api.api.rest.base.serializers import validate_json
+from metax_api.models import CatalogRecord
+from metax_api.tests.utils import test_data_file_path, get_json_schema, TestClassUtils
+
+
+schema = get_json_schema('ida_dataset')
+
+
+class ValidateJsonTests(APITestCase, TestClassUtils):
+
+    """
+    Test some customizations of the json schema validator.
+    """
+
```

Comment:
The presence of these tests in this PR is a cockup. Should have been included in this PR https://github.com/CSCfi/metax-api/pull/343